packages/superset-ui-query/src/buildQueryObject.ts

```diff
@@ -24,23 +25,25 @@ export default function buildQueryObject<T extends QueryFormData>(formData: T):
     time_range,
     since,
     until,
-    columns = [],
-    groupby = [],
     order_desc,
     row_limit,
     limit,
     timeseries_limit_metric,
+    controlGroups,
+    ...residualFormData
   } = formData;
 
+  const groupedControls = buildGroupedControlData(residualFormData, controlGroups);
+  const { metrics, groupby, columns } = groupedControls;
```

Comment:
nit

```ts
const { metrics, groupby, columns } = buildGroupedControlData(residualFormData, controlGroups);
```

Reply:
Is anything from the object returned by `buildGroupedControlData` expected to be useful besides `metrics, groupby, columns`? If not, it may be possible to simplify that method's logic to ignore the extraneous stuff.